Data-Standards/Patch-Seq/electrophysiology-metadata-schema.json

```diff
@@ -0,0 +1,145 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "electrophysiology-metadata-schema.json",
+  "title": "Intracellular electrophysiology metadata",
+  "description": "Formal description of the IC Ephys Metadata List",
+  "author": "Ben Dichter",
+  "contact": "ben.dichter@gmail.com",
+  "version": "0.1.0",
+  "type": "object",
+  "required": [
+    "species",
+    "subject id",
+    "subject age",
+    "subject sex",
+    "subject date of birth",
```

Comment:
Many groups will not be able to supply this, what is the failure mode if they cannot and this is required? `N/A`?

Reply:
One thing that might be possible to require that all groups could supply is age category.

Reply:
The point of this document is to guide researchers to what data they should record (and to come to a common consensus about it). Maybe we don't have date_of_birth for old data, but we should be able to collect it for future sessions. 

I don't really like the idea of filler for required fields. We have optional fields and if we want to make this optional we can do that. But whatever fields we mark as required, there will of course be datasets that are missing it, and what to do in those cases is more of a policy question. I prefer having minimal rules and enforcing them strictly rather than having strict rules and enforcing them leniently, but there will probably be a mix of both. Ultimately, we'll need to come up with a strategy that promotes adherence to these rules with minimal hindering of data sharing.

I do think it is a bit redundant to have subject_age and subject_date_of_birth.

What do you mean by age category? Is that something we can standardize across groups?